creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.js

```diff
@@ -62,14 +74,21 @@ module.exports = async function ({ logger, syncType, syncRequestParameters }) {
     return {
       error: {
         message: errorMsg
-      }
+      },
+      jobsToEnqueue: {}
+    }
+  }
+  if (syncMode === SYNC_MODES.None) {
```

Comment:
just wondering if it's intentional to not log a metric here. I noticed the above metric values allows SyncMode=None, but maybe it's not worth storing this data since it could indicate an inefficiency but no real issue if we're attempting a bunch of no-ops?

Reply:
good question here and below - will answer both here
i wasn't really sure how to record a metric, unless you meant literally to log it? very down to log it here, like i have below

unlike the `FIND_SYNC_REQUEST_COUNTS_GAUGE`, i didn't see any metric that i could record, since `ISSUE_SYNC_REQUEST_MONITORING_DURATION_SECONDS_HISTOGRAM` is only for actually issuing the sync with label `reason_for_additional_sync` which doesn't apply here

i am def down to update this to be similar to the `FIND_SYNC_REQUEST_COUNTS` and record all outcomes? wdyt

Reply:
hmm I guess maybe we can wait til there's a clear need for another metric but I was thinking the biggest use of a metric here would be to know if sync logic is being short circuited before reaching the part that records the `ISSUE_SYNC_REQUEST_MONITORING_DURATION_SECONDS_HISTOGRAM` metric. or if there are any errors in the `syncRequestParametersForceResync` axios request on line 172 or the `syncRequestParameters` axios request on line 174. I don't feel too strongly either way tho -- up to you